src/backend/api/main.py

```
@@ -0,0 +1,175 @@
+            "processedEmails": 0,
+            "eventsCreated": 0,
+            "status": f"error: {str(e)}"
+        }
+
+
+def _extract_job_opportunities(email: Dict) -> List[Dict[str, Any]]:
+    """Extract job opportunities from email content."""
+
+    opportunities = []
+    subject = email.get('subject', '')
+    body = email.get('body', '')
```

Ruff (0.12.2)

Local variable `body` is assigned to but never used

Remove assignment to unused variable `body`

(F841)